superchain/configs/mainnet/op.yaml

```diff
@@ -1,4 +1,5 @@
-name: OP-Mainnet
+name: OP Mainnet
```

Comment:
We should verify if these changes will break something in the downstream dependencies (or even somewhere in this repo that I have missed).

Reply:
https://github.com/ethereum-optimism/optimism/pull/10043

Reply:
https://github.com/ethereum-optimism/op-geth/pull/287